docs/source/contributing/testing.rst

```diff
@@ -167,69 +166,44 @@ For example, to test the ``Circle`` VMobject which resides in
 ``manim/mobject/geometry.py``, add the CircleTest to
 ``test/test_geometry.py``.
 
-In ``test_geometry.py``, add:
-
-.. code:: python
+You will need to use the ``frames_comparison`` decorator to create a test. The test function **must** take as a parameter a variable named ``scene`` that will be used like ``self`` in a standard ``construct`` method.
```

Comment:
This seems important enough that it should be an admonition.

+a clarity suggestion
```suggestion
.. important::
    You will need to use the ``frames_comparison`` decorator to create a test. The test function **must** accept a 
    parameter named ``scene`` that will be used like ``self`` in a standard ``construct`` method.
```